articles/azure-portal/azure-portal-safelist-urls.md

```diff
@@ -80,6 +80,7 @@ api.aadrm.com (Microsoft Entra)
 api.loganalytics.io (Log Analytics Service)
 api.azrbac.mspim.azure.com (Microsoft Entra)
 *.applicationinsights.azure.com (Application Insights Service)
+appmanagement.activedirectory.microsoft.com (Microsoft Entra)
```

Comment:
looks like only the public cloud was updated. Wasn't the intention to also add mc and usgov lists?

Reply:
*.msidentity.us and *.msidentity.cn cover the national cloud scenarios. Specifically, appmanagement.msidentity.us and appmanagement.msidentity.cn.